examples/Components/linearsolver/FEMBAR-MinResLinearSolver.scn

```diff
@@ -13,9 +13,10 @@
         <EulerImplicitSolver name="odesolver" rayleighStiffness="0.1" rayleighMass="0.1" />
         <MinResLinearSolver template="GraphScattered" iterations="1000" tolerance="1e-9" />
         <MechanicalObject />
-        <UniformMass vertexMass="1" />
+        <UniformMass totalMass="320" />
```

Comment:
With `vertexMass`, if you change the resolution of the grid, it will also change the total mass of the object. Not with `totalMass`.
Here, there are 320 vertices, so the total mass is 320. If I change the grid resolution, I still want a total mass of 320.